src/main/java/com/timgroup/statsd/Event.java

```diff
@@ -81,8 +81,8 @@ public Event build() {
             if ((event.title == null) || event.title.isEmpty()) {
                 throw new IllegalStateException("event title must be set");
             }
-            if ((event.text == null) || event.text.isEmpty()) {
-                throw new IllegalStateException("event text must be set");
+            if (event.text == null) {
+                event.text = new String("");
```

Comment:
Why not leave it null here?

Reply:
I agree, this would be ideal. You'd need further changes to avoid triggering `NullPointerException`. Your test below should catch and highlight the necessary additional changes.